src/installer/pkg/projects/netcoreapp/src/netcoreapp.depproj

```diff
@@ -105,7 +100,7 @@
 
     <ItemGroup Condition="'$(PackageRID)' != ''">
       <LibrariesRuntimeFiles Condition="'%(IsNative)' != 'true'" TargetPath="runtimes/$(PackageRID)/lib/$(NetCoreAppCurrent)" />
-      <LibrariesRuntimeFiles Condition="'%(IsNative)' == 'true'" TargetPath="runtimes/$(PackageRID)/native/%(RecursiveDir)" />
+      <LibrariesRuntimeFiles Condition="'%(IsNative)' == 'true'" TargetPath="runtimes/$(PackageRID)/native/%(LibrariesRuntimeFiles.NativeSubDirectory)%(RecursiveDir)" />
```

Comment:
Took me a while to understand what this is doing. You might want to add a comment.

Reply:
Yeah though that applies to all of this build logic, it's pretty hard to understand without spending a lot of time looking at build flows in the binlog. 😄

Reply:
Will merge to unblock wasm and add the comment in another PR.